### DISCUSSION TITLE
Number of seats

### DISCUSSION THREAD

**Discussion by amsilva84** [liedekef/events-made-easy#319] [Q&A] [answered]
Hello,

how do I hide the seats row from the table eme-rsvp-form?  the max amount of seats is always one. I tried using the tr hidden="hidden" but it does not work inside the booking form format

**Comment by refugeindhamma** [comment, marked as answer]
I would suggest `style="display:none;"`

**Reply by amsilva84** [reply]
awesome, thank you. i did use this with last-child to hide it

**Comment by liedekef** [comment]
If you set the min and max number of seats to book to the same number (like "1"), it will be hidden automatically by EME.

**Reply by amsilva84** [reply]
Hello Franky, 
thats how its setup, both max and min of seats its setup for 1 but the seats row shows up although there is no option to select any which is good 
![seats](https://github.com/liedekef/events-made-easy/assets/146024289/f020ce61-fc3d-4e44-baa5-df1f23226fcd)

**Reply by liedekef** [reply]
The "row" is your fom layout. Feel free to change the form and put #_SEATS outside the table (so no row will show).

**Reply by amsilva84** [reply]
oh ok, ill give that a try! Thanks!!
